fullcase.js

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env node
+
+const fs = require("fs");
+
+const SpecialCasing = ""+fs.readFileSync("SpecialCasing.txt");
+const CaseFolding = ""+fs.readFileSync("CaseFolding.txt");
+
+const special =
+  /^([0-9A-F]+);\s+([0-9A-F]+(?:\s+[0-9A-F]+)*)?;\s+([0-9A-F]+(?:\s+[0-9A-F]+)*)?;\s+([0-9A-F]+(?:\s+[0-9A-F]+)*)?;\s+(#.+?)?$/mg;
+
+const folding =
+  /^([0-9A-F]+);\s+([CFS])?;\s+([0-9A-F]+(?:\s+[0-9A-F]+)*)?;\s+(#.+?)?$/mg;
+
+const scData = {};
+const cfData = {};
+
+const rhex = a => typeof a === 'string' ? +('0x'+a) : +a;
```

Comment:
```suggestion
const rhex = a => parseInt(a, 16);
```

Do we expect any of these keys to not be strings?